src/test/java/DukeTest.java

```diff
@@ -0,0 +1,20 @@
+import duke.Duke;
+import duke.parser.Parser;
+import duke.task.Task;
+import duke.task.Todo;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DukeTest {
+    @Test
+    public void testTaskSizeParser() {
```

Comment:
Perhaps a more descriptive name for the test function?

```suggestion
    public void testTaskSizeParser_emptyString() {
```